addons/l10n_it_edi/models/res_partner.py

```diff
@@ -148,3 +145,60 @@ def validate_codice_fiscale(self):
         for record in self:
             if record.l10n_it_codice_fiscale and (not codicefiscale.is_valid(record.l10n_it_codice_fiscale) and not iva.is_valid(record.l10n_it_codice_fiscale)):
                 raise UserError(_("Invalid Codice Fiscale '%s': should be like 'MRTMTT91D08F205J' for physical person and '12345670546' for businesses.", record.l10n_it_codice_fiscale))
+
+    def action_open_partners_l10n_it_edi(self, res_ids=None, title=None):
+        if len(res_ids) == 1:
+            return self.action_open_partners(res_ids, title)
+        return {
+            'type': 'ir.actions.act_window',
+            'res_model': 'res.partner',
+            'view_mode': 'tree',
+            'views': [(self.env.ref('l10n_it_edi.res_partner_tree_l10n_it').id, 'list')],
```

Comment:
will this work if the module wasn't updated?
I don't think the view will exist

Reply:
If the module isn't updated, the widget will not show the links, therefore -> no action.